### ISSUE TITLE
Maybe we need an option to set the browser restarted between test suite after version 1.0.

### ISSUE THREAD

**Issue by junguchina** [Codeception/CodeceptJS#628]
When I upgrade the codeceptjs to version 1.0.0,I found the new change like below:
* Don't close browser between suites, when restart option is false. We should start browser only one time and close it only after all tests.

This change makes my test suite failed, before the version 1.0.0, the browser will restart and clean the cookie between the test suite.
And in my test suites, some suites need to login the account but others need not. So with version 1.0.0, the browser will not restart anymore between test siutes.
I think lot of guys will have same problem like me, some test suite need the cookie but others needn't.
If don't close browser between suites, the cookie will always in the browser.
So I suggest, maybe we need an option to set the browser restarted between test suite after codeceptjs 1.0.
@APshenkin Could you have a look on this feature request？

**Comment by APshenkin** [comment]
@junguchina 
Can you please provide more information?
We clear cookies and local storage between each test, when `restart:false`. Also there is an option `keepCookies` that clear only local storage between test

`If don't close browser between suites, the cookie will always in the browser.` - Please provide more information about this issue

So now you can combine flags to get what you want.
1 option - clear cookies between each test when `restart:false`
2 option - don't clear cookies between each test (`keepCookies`) and clear them exact from your tests or in `Before` hook

BTW I recommend to clear cookies between each test and add auth cookie before each test to provide more idempotency in your tests

**Comment by DavertMik** [comment]
Related #629

**Comment by junguchina** [comment]
@APshenkin Pls refer my config in codecept.conf.js like below:
```

**Comment by APshenkin** [comment]
@junguchina can you please provide more information about test cases? or give a link for your web app?

I think you can do this trick
Set `restart:false`, remove `keepCookies` option
in suite A add `Before` hook, where you can set login cookie before each test

**Comment by APshenkin** [comment]
BTW, please check https://github.com/Codeception/CodeceptJS/issues/631
Maybe the reason of your problem is the same

**Comment by junguchina** [comment]
Hi @APshenkin ,
I referred the issue #631 and check my cookies in browser, the cookies are stored in same domain between test suites.So I think this is not same with #631.
You can refer the log with tag `--verbose` between test siuts:
```

**Comment by junguchina** [comment]
Hi @APshenkin ,
I referred the #631 and checked our web app with developers.
I found that we changed the login mechanism and used the [node openid](https://github.com/panva/node-openid-client) to provide account auth.
There is a auth server named OP and our web app named RP, the codeceptjs will cleared the cookies of RP, but the authentic info still stored in OP, so between test suites, the RP still can pass the auth.

Now I add a login case in first and a logout case in latest of the test suite which needs to login account.
This can make sure the single test suite and all test suites passed with codeceptjs.

Thank you @APshenkin for your help, pls close this issue.
